### PR TITLE
fixed button issue on card list

### LINKED ISSUE #331: Fix card list button issue
Card list buttons are not wokring , as show in below . The fix is should make buttons visible inside card list .

Image

### PR DESCRIPTION
**Pull request by sanmishr**
Fixed button style issues 

Fix #331 

Test URLs:
- Before: https://main--hlx-test--urfuwo.hlx.live/topics/
- After: https://331-card-list-button--hlx-test--urfuwo.hlx.page/topics/

### CODE REVIEW

File: aemedge/styles/styles.css [hunk 1]
```diff
@@ -12,33 +12,33 @@
 /* Fall back to Arial for 72 Brand Variable
  Added to styles.css for early loading */
 
-@font-face {
-  font-family: '72 Brand Variable Fallback';
+ @font-face {
+  font-family: "72 Brand Variable Fallback";
```
Comment (mhaack) [inline aemedge/styles/styles.css:16]: Be careful with these, we should use all the same format rules.

File: aemedge/styles/styles.css [hunk 2]
```diff
@@ -194,11 +194,21 @@ a:hover {
...
+  em {
     display: block;
   }
-  & +p {margin-top: 0;}
 
+  & + p {
+    margin-top: 0;
+  }
+
+  a.button {
```
Comment (mhaack) [inline aemedge/styles/styles.css:206]: This should be the same as ` .button` below, right?

**Review by mhaack — COMMENTED** [review]
The buttons look ok but there is now hover effect

File: aemedge/styles/styles.css (hunk 1 above)
Comment (sanmishr) [reply aemedge/styles/styles.css]: followed existing indentation

File: aemedge/styles/styles.css (hunk 2 above)
Comment (sanmishr) [reply aemedge/styles/styles.css:205]: right added hover effect as well from below

File: aemedge/styles/styles.css [hunk 3]
```diff
@@ -12,7 +12,7 @@
 /* Fall back to Arial for 72 Brand Variable
  Added to styles.css for early loading */
 
-@font-face {
+ @font-face {
```
Comment (mhaack) [inline aemedge/styles/styles.css:15]: Please revert that format change

File: aemedge/styles/styles.css (hunk 2 above)
Comment (mhaack) [reply aemedge/styles/styles.css:205]: > This should be the same as  .button below, right?

... so why do we need .button and a.button?

File: aemedge/styles/styles.css (hunk 3 above)
Comment (sanmishr) [reply aemedge/styles/styles.css:15]: reverted

File: aemedge/styles/styles.css (hunk 2 above)
Comment (sanmishr) [reply aemedge/styles/styles.css:205]: for the very same reason why it was not populating initially . I.e 

image

vs 

image

hence the anchor tags need styling of its own .

**Review by sanmishr — COMMENTED** [review]
updated reviews

File: aemedge/styles/styles.css (hunk 2 above)
Comment (sanmishr) [reply aemedge/styles/styles.css:202]: Because the anchor tags are a direct child in this case . They need special styling .

**Comment by sanmishr** [comment]
@mhaack  closing this ticket as this is fixed along with content fixes

**Pull request closed without merging**